package.json

```diff
@@ -15,6 +15,7 @@
     "lint": "kcd-scripts lint",
     "test": "kcd-scripts test",
     "test:cover": "kcd-scripts test --coverage",
+    "test:native": "kcd-scripts test --config other/react-native/jest.config.js --rootDir .",
```

Comment:
Additionally, doing some more reading around Jest and what people use for configs, using a config file combined with `--rootDir .` was the most flexible way to set this up:
https://github.com/facebook/jest/issues/2670
https://github.com/facebook/jest/issues/3613